src/stylua.ts

```diff
@@ -13,24 +14,27 @@ interface GitHubRelease {
 
 async function getReleases(token: string): Promise<GitHubRelease[]> {
   const octokit = getOctokit(token)
-  const {data: releases} = await octokit.repos.listReleases({
+  let {data: releases} = await octokit.repos.listReleases({
     owner: 'JohnnyMorganz',
     repo: 'stylua'
   })
 
+  // Don't include drafts and pre-release
+  releases = releases.filter(r => !(r.draft || r.prerelease))
   // Sort by latest release first
   releases.sort((a, b) => semver.rcompare(a.tag_name, b.tag_name))
   return releases
 }
 
-function getLatestVersion(releases: GitHubRelease[]): string | null {
-  return semver.clean(releases[0].tag_name)
-}
-
 function chooseRelease(
   version: string,
   releases: GitHubRelease[]
 ): GitHubRelease | undefined {
+  if (version === '') {
+    core.debug('No version provided, finding latest release version')
+  } else if (version === 'latest') {
+    version = ''
+  }
   return releases.find(release => semver.satisfies(release.tag_name, version))
```

Comment:
I saw in some places you were `semver.clean()`ing the release.tag_name, which I think makes sense. Just confirming that it looks like it's not necessary with `.satisfies()` (as you already had in your code):

```
> semver.satisfies("0.17.1  ", "0.17")
true
```